src/embeddings/tools/embedding_vocab.cpp

```diff
@@ -16,6 +16,28 @@
 
 using namespace meta;
 
+namespace
+{
+    const std::string SGNS  = "sgns";
+    const std::string GLOVE = "glove";
```

Comment:
The topic modeling portion of the library actually does this as well. That said, I'm open to suggestions.

I also could look at why we even did this. I recall it was due to some differences during vocabulary generation. It may not have even been worth it.